docs/kola/external-tests.md

```diff
@@ -103,6 +103,52 @@ it out.
 (Previously the API for this was to send `SIGTERM` to the current process; that
 method is deprecated and will be removed at some point)
 
+## HTTP Server
+
+The `kolet` binary is copied into the CoreOS system running the tests. Notably,
+it contains the built-in command `kolet httpd [port] [path]` for starting an
+HTTP file server at port to serve the contents of the file system rooted at path.
+
+To have an HTTP server started before your tests runs, you can create an Ignition
+config (or Fedora CoreOS config) and include it in your external test directory.
```

Comment:
This is definitely a correct approach, but it feels heavyweight.  Contrast with e.g. the ultra low tech approach of just doing: 
```
echo somedata > /var/home/core/somedata.txt
kolet httpd 8080 /var/home/core &
```
in a test script.

There are various problems with `&` around having it be racy when the process is listening, and correctly checking for errors - but it is a convenient way to do this.

Reply:
ah right, this would be more conveninent. I didn't realize we can add that `&`.